src/vivarium_ciff_sam/components/lbwsg.py

```diff
@@ -43,7 +43,16 @@ def calculate_csmr(cause: str, builder: Builder) -> pd.DataFrame:
         )
 
 
-class LBWSGRisk(Risk, ABC):
+class LBWSGRisk(Risk):
+
+    def __init__(self):
+        super().__init__('risk_factor.low_birth_weight_and_short_gestation')
+
+    def _get_population_view(self, builder: Builder) -> PopulationView:
+        return builder.population.get_view([self.propensity_column_name, 'tracked'])
+
+
+class LBWSGSubRisk(Risk, ABC):
```

Comment:
Im not understanding what this refactoring did here.  It added a class instance of Risk with LBWSGRisk along with an ABC in LBWSGSubRisk instead of just having a ABC.  Is this more future proofing since the ABC is still the one being referenced later (lines 142, 156, etc).

Reply:
the first is the categorical risk, and the second is an abstract base class for the continuous risk classes for birth-weight and gestational age